.eslintrc.js

```diff
@@ -27,7 +27,17 @@ module.exports = {
     // react/jsx-uses-react and react/react-in-jsx-scope rules are no longer necessary and can be turned off or removed.
     "react/jsx-uses-react": "off",
     "react/react-in-jsx-scope": "off",
-    "@typescript-eslint/no-unused-vars": "error",
+    "@typescript-eslint/no-unused-vars": [
+      "error",
+      {
+        argsIgnorePattern: "_",
+        varsIgnorePattern: "_",
+      },
+    ],
+    "@typescript-eslint/no-empty-function": [
+      "error",
+      { allow: ["arrowFunctions", "methods", "asyncMethods"] },
+    ],
```

Comment:
The rules have been loosened slightly to take into account the following areas.

> varsIgnorePattern: "_",

https://github.com/monstar-lab-oss/reactjs-boilerplate/blob/bf7ceea75a926b701adba750bd732aac90ae41a5/src/hooks/useAuth.tsx#L32

> allow: ["arrowFunctions", "methods", "asyncMethods"]

https://github.com/monstar-lab-oss/reactjs-boilerplate/blob/bf7ceea75a926b701adba750bd732aac90ae41a5/src/hooks/useAuth.tsx#L24-L25